### PR TITLE
register login screen

### LINKED ISSUE #7: Register Or Login Screen

### PR DESCRIPTION
**Pull request by VariantAbdulaziz**

### CODE REVIEW

File: qelem/lib/screens/register_login_screen.dart [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+class RegisterLoginScreen extends StatelessWidget {
+  const RegisterLoginScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final pads = MediaQuery.of(context).size.height / 15;
+    final buttonHeight = MediaQuery.of(context).size.height / 15;
+    final imageHeight = MediaQuery.of(context).size.height / 2;
+    final mainWelcomeTextStyle = Theme.of(context).textTheme.titleMedium;
```
Comment (brukted) [inline qelem/lib/screens/register_login_screen.dart]: it is headline 5

File: qelem/lib/screens/register_login_screen.dart [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    final additionalIntrosStyle = Theme.of(context).textTheme.subtitle1;
```
Comment (brukted) [inline qelem/lib/screens/register_login_screen.dart]: it is body 2

File: qelem/lib/screens/register_login_screen.dart [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                width: double.infinity,
+                height: buttonHeight + 2 * pads,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceAround,
+                  children: [
+                    ElevatedButton(
+                      style: ElevatedButton.styleFrom(
+                        primary: Theme.of(context).primaryColor,
+                        minimumSize: Size(double.infinity, buttonHeight),
+                      ),
```
Comment (brukted) [inline qelem/lib/screens/register_login_screen.dart]: Just use the default style, there is nothing special about these buttons. We will customize them from the material app theme. The same goes for the other button.

**Review by brukted — CHANGES_REQUESTED** [review]

File: qelem/lib/screens/register_login_screen.dart (hunk 1 above)
Comment (VariantAbdulaziz) [reply qelem/lib/screens/register_login_screen.dart]: the mainwelcometextstyle?

File: qelem/lib/screens/register_login_screen.dart (hunk 2 above)
Comment (VariantAbdulaziz) [reply qelem/lib/screens/register_login_screen.dart]: will do.

File: qelem/lib/screens/register_login_screen.dart (hunk 3 above)
Comment (VariantAbdulaziz) [reply qelem/lib/screens/register_login_screen.dart]: what do you mean the default style?

File: qelem/lib/screens/register_login_screen.dart (hunk 1 above)
Comment (brukted) [reply qelem/lib/screens/register_login_screen.dart]: yes

**Comment by brukted** [comment]
add the image to assets in the pubspec.yaml

File: qelem/lib/screens/register_login_screen.dart (hunk 3 above)
Comment (brukted) [reply qelem/lib/screens/register_login_screen.dart]: The style from the material app theme, that is inherited by default.

**Review by Biruk-Tassew — APPROVED** [review]

**Review by nawrazi — APPROVED** [review]

**Review by brukted — APPROVED** [review]

**Review by hello-world-was-taken — APPROVED** [review]

**Pull request merged**